### DISCUSSION TITLE
Mobile use case instead of SQLite

### DISCUSSION THREAD

**Discussion by likern** [duckdb/duckdb#2212] [Q&A]
Hello 👋 
Given that all cool and modern features are applied (like ART indexes), it's multithreaded / vectorized and embedded...
Can it be used in usual OLAP workloads as usual main database for mobile applications?

Is there some kind of benchmark to look at?

**Comment by hannes** [comment]
I am certain it can be used for this use case, it was also designed to to well there. Curious about some feedback though.

**Reply by likern** [reply]
Thanks, will try it out to integrate.
